scripts/o3de/CMakeLists.txt

```diff
@@ -7,3 +7,23 @@
 #
 
 add_subdirectory(tests)
+
+ly_get_list_relative_pal_filename(pal_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/${PAL_PLATFORM_NAME})
+include(${pal_dir}/o3de_install_${PAL_PLATFORM_NAME_LOWERCASE}.cmake)
```

Comment:
If configuring for Android or iOS, then these files will be missing and the include command will fail causing a configure error

Reply:
this one is handled in `scripts/CMakeLists.txt`, there is a `if(PAL_TRAIT_BUILD_HOST_TOOLS)` there

Reply:
Ah that covers it. Thanks for the information